common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -39,6 +39,38 @@ components:
       pattern: ^[0-9]{20}$
       description: |
                     Het Organisatie-identificatienummer (OIN) is een uniek nummer dat Logius kan toekennen aan organisaties om zich te kunnen identificeren, authentiseren en of autoriseren bij digitaal berichtenverkeer binnen en met de overheid.
+    Point:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+    LineString:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/linestringGeoJSON.yaml'
+    Polygon:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+    MultiPoint:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multipointGeoJSON.yaml'
+    MultiLineString:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multilinestringGeoJSON.yaml'
+    MultiPolygon:
+      $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multiPolygonGeoJSON.yaml'
+    GeoJSONGeometrie:
+      title: GeoJSON Geometry
+      description: Geometrie in GeoJSON formaat (RFC 7946). Een property die dit datatype heeft mag slechts 1 van de opgenomen properties bevatten. Deze constructie vervangt de oneOf vanwege codegeneratie-issues.
+      externalDocs:
+        description: RFC 7946 - The GeoJSON Format
+        url: 'https://tools.ietf.org/html/rfc7946'
+      type: object
+      properties:
+        point:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+        lineString:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/linestringGeoJSON.yaml'
+        polygon:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
+        multiPoint:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multipointGeoJSON.yaml'
+        multiLineString:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multilinestringGeoJSON.yaml'
+        multiPolygon:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/multiPolygonGeoJSON.yaml'
```

Comment:
Bovenstaande lezend realiseer ik me dat deze vorm vermoedelijk problemen op gaat leveren.

Als er vanuit een yaml bestand naar bijv. GeoJSONGeometrie wordt verwezen is er geen probleem. Een yaml bestand verwijst naar meerdere andere yaml bestanden.
Echter als we het JSON bestand willen gebruiken dan staat ook daarin een verwijzing naar yaml bestanden wat betekent dat je uiteindelijk json en yaml door elkaar heen krijgt. 
Misschien kan een JSON parser daarmee omgaan maar ik vermoed van niet.

Reply:
Hebben we besproken.